src/Microsoft.OData.Client/Serialization/HttpClientRequestMessage.cs

```diff
@@ -89,6 +91,24 @@ public HttpClientRequestMessage(DataServiceClientRequestMessageArgs args)
             }
         }
 
+#if NETSTANDARD2_0_OR_GREATER
+        /// <summary>
+        /// Gets the underlying <see cref="HttpClientHandler.ServerCertificateCustomValidationCallback"/> so that callers can customize their certificate valiation

```

Comment:
Gets / sets...,

Reply:
+1

Reply:
```suggestion
        /// Gets or sets the underlying <see cref="HttpClientHandler.ServerCertificateCustomValidationCallback"/> so that callers can customize their certificate valiation
```